pkg/kindsys/schema_interface.cue

```diff
@@ -90,7 +90,7 @@ SchemaInterface: {
 // The canonical list of all Grafana schema interfaces.
 schemaInterfaces: [N=string]: SchemaInterface & { name: N }
 schemaInterfaces: {
-	Panel: {
+	PanelCfg: {
```

Comment:
This PR changes the names of the schema interfaces - formerly known as "slots" - to these new names:

* `Panel` -> `PanelCfg`
* `Query` -> `Queries`
* `DSOptions` -> `DatasourceCfg`

Talking about "the panel slot" was really ambiguous with "panel" ([the base type](https://github.com/grafana/grafana/blob/9cdcbab2fc3b5508077cb5e9715bcf7d99c7c4d8/kinds/dashboard/dashboard_kind.cue#L333) defined in the dashboard schema) and with panel plugins themselves. While more verbose, i think "the panelcfg schema interface" is a significant improvement.

We haven't been using the other two yet. I still have some questions about `Queries` that i hope to tackle in the coming weeks, but i think that the similarity of names between `DatasourceCfg` with `PanelCfg` will be a useful pattern to follow.

Reply:
Nit. Maybe `PanelOptions` is more suitable given https://grafana.com/docs/grafana/next/panels-visualizations/configure-panel-options/ ? Or instead of `Cfg` use `Config` of `Configuration`

Reply:
Sorry I just realize that `PanelOptions` is a child property of `PanelCfg`